docker-compose.yml

```diff
@@ -150,7 +150,7 @@ services:
     volumes:
       - ./config/form-content:/config
   mandataris:
-    image: lblod/mandataris-service:0.0.8
+    image: lblod/mandataris-service:0.0.9
```

Comment:
Where do you get 0.0.9 from?
https://hub.docker.com/r/lblod/mandataris-service/tags

package.json is on 0.0.3 😆